base_menu_visibility_restriction/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This addon adds "Restricted groups" in Menu items.
```

Comment:
An alternative text suggestion, if it's clearer:
This addon lets you assign "excluded groups" to menu items. If a user belongs to a group that is assigned to a menu item as an excluded group, the user will not be able to see the menu item.